app/api/v1/endpoints/admin.py

```
@@ -738,31 +812,50 @@ async def get_audit_logs(
         
         if start_date:
-            start_dt = datetime.fromisoformat(start_date)
-            query = query.filter(AuditLog.created_at >= start_dt)
+            try:
+                start_dt = datetime.fromisoformat(start_date)
+                stmt = stmt.where(AuditLog.created_at >= start_dt)
+            except ValueError:
+                raise HTTPException(
+                    status_code=status.HTTP_400_BAD_REQUEST,
+                    detail=f"Invalid date format for start_date: {start_date}. Expected ISO format (YYYY-MM-DD or YYYY-MM-DDTHH:MM:SS)"
+                )
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)